tests/infer/mcmc/test_mcmc.py

```diff
@@ -104,8 +102,7 @@ def _empty_model():
 @pytest.mark.parametrize("jit", [False, True])
 @pytest.mark.parametrize("num_chains", [
     1,
-    skipif_param(2, condition="CI" in os.environ or "CUDA_TEST" in os.environ,
-                 reason="CI only provides 1 CPU; also see https://github.com/pytorch/pytorch/issues/2517")
```

Comment:
CUDA_TEST pass in my system

Reply:
I am still seeing CUDA initialization error for this test with `CUDA_TEST=1 PYRO_TENSOR_TYPE=torch.cuda.DoubleTensor pytest -vs -k test_empty_sample_sites` with the latest pytorch.

Reply:
Sorry, you are right. That is my mistake. I did use only `CUDA_TEST=1`. We'd need `PYRO_TENSOR_TYPE=torch.cuda.DoubleTensor` too. >"<

Reply:
I'll revert it back in the last PR.